cpp/lib/include/FileUtil.h

```diff
@@ -270,6 +270,13 @@ inline std::string MakeAbsolutePath(const char * const relative_path)
 bool MakeEmpty(const std::string &path);
 
 
+/** \brief Create an empty file or change last modified timestamp of an existing file. */
+void TouchFileOrDie(const std::string &path);
+
+
+void ChangeOwnerOrDie(const std::string &path, const std::string &user, const std::string &group="", bool recursive=true);
```

Comment:
I'd make the "user" argument optional, too.  That way you can change either the owner or the group, or both.
"recursive" should be const, too!